py-polars/pypolars/frame.py

```diff
@@ -86,11 +86,11 @@ def from_parquet(
         DataFrame
         """
         self = DataFrame.__new__(DataFrame)
-        self._df = PyDataFrame.from_parquet(file, batch_size)
+        self._df = PyDataFrame.read_parquet(file, batch_size)
```

Comment:
I think this should remain `from_parquet`, or we should also change it on the Rust side.